crates/orchestrator/src/tests/jobs/mod.rs

```diff
@@ -180,7 +184,7 @@ async fn process_job_with_job_exists_in_db_and_valid_job_processing_status_works
 
     // Queue checks
     let consumed_messages =
-        services.config.queue().consume_message_from_queue(JOB_VERIFICATION_QUEUE.to_string()).await.unwrap();
+        services.config.queue().consume_message_from_queue(verification_queue.to_string()).await.unwrap();
```

Comment:
Queue_Name can be used here.

Reply:
It cant be used here, its a case value from the args.

Reply:
My bad we can use it, updated